pkg/app/piped/executor/waitapproval/waitapproval.go

```diff
@@ -54,8 +54,8 @@ func (e *Executor) Execute(sig executor.StopSignal) model.StageStatus {
 		ticker         = time.NewTicker(5 * time.Second)
 	)
 	defer ticker.Stop()
-
-	timer := time.NewTimer(time.Duration(e.StageConfig.WaitApprovalStageOptions.Timeout) * time.Second)
```

Comment:
just to confirm, the bug caused by this deleted `time.Second` annotation, is that right? 🤔 
(Timeout default value currently set as 6h already ref: [config/deployment](https://github.com/pipe-cd/pipe/blob/master/pkg/config/deployment.go#L25))

Reply:
I think so. It always overflows and looks like being `time.NewTimer(-315919h31m53.884942336s)` at runtime.
https://play.golang.org/p/WhPnXvUp7Ci

Reply:
I see, I met this issue before 😂 ( time.Minute * time.Minute overflow in my case )

Reply:
And one more minor question, it looks like the default value is of type Duration already, is that `time.Duration(...)` is necessary for the new code? 👀

Reply:
Look closely. The type of `Duration` used by Config is our own Type for making it easier to parse. So we have to parse it to time.Duration when using 😄

Reply:
Yeh, but it a defined type based on `time.Duration` (ref: [config/duration.go](https://github.com/pipe-cd/pipe/blob/master/pkg/config/duration.go#L23) ). Still looks weird to me, but never mind if it works that way 😅

Reply:
I mean
```go
e.StageConfig.WaitApprovalStageOptions.Timeout.Duration()
```
should be fine 😂

Reply:
Ah, you're right, let me fix.